index.html

```diff
@@ -71,13 +70,13 @@
   </div>
 
   <footer class="content">
-    <nav>
-      <a href="https://github.com/iojs/io.js/issues">GitHub Issues</a><!--
-   --><a href="https://github.com/iojs">GitHub Org</a><!--
-   --><a href="https://webchat.freenode.net/?channels=io.js">IRC Chat</a><!--
-   --><a href="http://logs.libuv.org/io.js/latest">Logs</a><!--
-   --><a href="https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme">Governance</a>
-    </nav>
+    <div class="nav">
+      <a href="https://github.com/iojs/io.js/issues">GitHub Issues</a>
+      <a href="https://github.com/iojs">GitHub Org</a>
+      <a href="https://webchat.freenode.net/?channels=io.js">IRC Chat</a>
+      <a href="http://logs.libuv.org/io.js/latest">Logs</a>
+      <a href="https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme">Governance</a>
+    </div>
```

Comment:
These comments are here for a reason so that no character is registered between the tags. (Otherwise it creates spaces.)